TRB_label_assist/regenerate_classification_performance_results.py

```diff
@@ -15,10 +16,11 @@
 labeled_trip_df_map = {}
 expanded_labeled_trip_df_map = {}
 expanded_all_trip_df_map = {}
+ct_entry={}
 for u in all_users:
     ts = esta.TimeSeries.get_time_series(u)
     ct_df = ts.get_data_df("analysis/confirmed_trip")
-
+    ct_entry[u]=eamtr._get_training_data(u,None)
```

Comment:
this will read twice from the database. It seems like you should be able to get_training_data and then use `_to_data_df` to convert the entries into a dataframe.

Reply:
fixed.